rust/kcl-lib/src/simulation_tests.rs

```diff
@@ -66,19 +67,12 @@ where
     settings.bind(f);
 }
 
-fn read<P>(filename: &str, dir: P) -> String
-where
-    P: AsRef<Path>,
-{
-    std::fs::read_to_string(dir.as_ref().join(filename)).expect("Failed to read file: {filename}")
-}
-
 fn parse(test_name: &str) {
     parse_test(&Test::new(test_name));
 }
 
 fn parse_test(test: &Test) {
-    let input = read(&test.entry_point, &test.input_dir);
+    let input = read_to_string(&test.entry_point).unwrap();
```

Comment:
Rust std functions don't report the file that failed to read. It's really annoying when it errors out saying "can't read file". Which file? So we had the helper to always report that. Would be nice to bring that back.

Reply:
oh fuck im sorry i was like "why is this here" and nukes it, yes i can!

Reply:
There's an RFC: https://github.com/rust-lang/rfcs/pull/3084

If I recall correctly, an objection was that it requires an allocation. But I haven't followed it.